From d15c15edfc4c2388e394708594359f14323d24c2 Mon Sep 17 00:00:00 2001
From: Magnus Damm <damm@opensource.se>
Date: Mon, 24 Feb 2014 14:52:12 +0900
Subject: ARM: shmobile: Move SYSC base variable to inside ifdefs

Move the rcar_sysc_base variable to inside #ifdefs to avoid
triggering build warnings in case PM or SMP is not selected.

Signed-off-by: Magnus Damm <damm@opensource.se>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit c4ca5d80e03559fd95c526ece5ce39fc732a2511)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/mach-shmobile/pm-rcar.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/mach-shmobile/pm-rcar.c b/arch/arm/mach-shmobile/pm-rcar.c
index 17225db09558..1f465a12d1b1 100644
--- a/arch/arm/mach-shmobile/pm-rcar.c
+++ b/arch/arm/mach-shmobile/pm-rcar.c
@@ -15,8 +15,6 @@
 #include <asm/io.h>
 #include <mach/pm-rcar.h>
 
-static void __iomem *rcar_sysc_base;
-
 /* SYSC */
 #define SYSCSR 0x00
 #define SYSCISR 0x04
@@ -35,6 +33,7 @@ static void __iomem *rcar_sysc_base;
 
 #if defined(CONFIG_PM) || defined(CONFIG_SMP)
 
+static void __iomem *rcar_sysc_base;
 static DEFINE_SPINLOCK(rcar_sysc_lock); /* SMP CPUs + I/O devices */
 
 static int rcar_sysc_pwr_on_off(struct rcar_sysc_ch *sysc_ch,
-- 
2.1.2

